src/library/entities/Library.java

```diff
@@ -17,65 +17,67 @@
 @SuppressWarnings("serial")

```

Comment:
I've noticed that there are double line spaces for each method seperator, do we want to use a single or double line space for these? cc: @haynescrossley

Reply:
I am not sure about this one.

Reply:
cc: @haynescrossley